corehq/apps/reports/standard/monitoring.py

```diff
@@ -337,6 +337,31 @@ def sort_column(self):
         else:
             return "landmark_%d" % (landmark,)
 
+    @property
+    def sort_key(self):
```

Comment:
This method is pretty similar to `sort_column`. Might be worth sharing the logic between them.